commands/mining_daemon_test.go

```diff
@@ -90,17 +93,13 @@ func TestMiningAddPieceAndSealNow(t *testing.T) {
 	_, err = minerNode.AddPiece(ctx, files.NewBytesFile([]byte("HODL")))
 	require.NoError(t, err)
 
-	// Since the miner does not yet have power, we still need the genesis node to mine
-	// the miner's commitSector and the submitPoSt messages
-	series.CtxMiningNext(ctx, 2)
-
 	// start sealing
 	err = minerNode.SealNow(ctx)
 	require.NoError(t, err)
 
 	// We know the miner has sealed and committed a sector if their power increases on chain.
-	// Wait up to 3 minutes for that to happen.
-	for i := 0; i < 180; i++ {
+	// Wait up to 5 minutes for that to happen.
```

Comment:
"minutes" looks incorrect, the sleep below is 1 second. "rounds" might be less brittle.